plugins/katello/3.12/release_notes/release_notes.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: plugins/katello/documentation
+title: Release Notes
+version: 3.12
+---
+
+# Katello Nightly () Release Notes
```

Comment:
Can you change this to (here and also in nightly release notes so we won't have to manually change it later)

```
# Katello {{ page.version }} Release Notes
```

Reply:
Also, by looking at the CHANGELOG you created on KATELLO-3.12 branch you should be able to whip up some release notes. Let me know if you need help with that